application/controllers/admin/UserManagement.php

```diff
@@ -959,7 +933,12 @@ public function _sendAdminMail($type = 'registration', $aUser, $newPassword = nu
         $mailer->Body = $body;
         $mailer->isHtml(true);
         $mailer->emailType = $emailType;
-        return $mailer->sendMessage();
+
```

Comment:
https://github.com/LimeSurvey/LimeSurvey/blob/409556bf2fc939f6b159c26f9d5f5a40cdde1328/application/core/LimeMailer.php#L540 already return a boolean, thencan return it ?

Reply:
We should return the $oMailer to handle error case in calling method?